WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -981,13 +979,12 @@ decisions = {
 			NOT = { religion_group = muslim }
 			OR = {
 				NOT = { religion_group = pagan_group }
-				religion = zun_pagan
-				religion = zun_pagan_reformed
 				AND = {
 					religion_group = pagan_group
-					realm_size = 80
+					realm_size = 20
```

Comment:
It serves as a long-term goal. A 80-sized realm is not much,20 is ridiculously small.